src/parserPages/PageParser.php

```diff
@@ -4,18 +4,23 @@
 
 class PageParser extends Parser
 {
-    public function getValuesInItem(string $link): array
+    public function getValuesInItem(string $link): ?array
     {
         $collection = [];
-        $xml = simplexml_load_file($link);
-        foreach ($xml->xpath('//item') as $item) {
-            $objectXmlModel = new XmlFeedModel();
-            $objectXmlModel->setLink($item->link->__toString());
-            $objectXmlModel->setDescription($item->description->__toString());
-            $objectXmlModel->setPubDate($item->pubDate->__toString());
-            $objectXmlModel->setTitle($item->title->__toString());
-            $collection []  = $objectXmlModel;
+        @$xml = simplexml_load_file($link);
+        $errorMassage = 'This link cannot be parsing';
+        if ($xml === false) {
+            return $errorMassage;
```

Comment:
Its logically wrong to return both array or strings in a function. 
Please also check variable spelling.